js/app.js

```diff
@@ -1,6 +1,36 @@
 /* setup your angular app here */
```

Comment:
Here is how I moved from one array to the next (to the fruit column):

    $scope.toFruit = function(idx){
        $scope.allFruit.push($scope.fruitAndVeg[idx]);
        $scope.fruitAndVeg.splice(idx, 1);
        if ($scope.fruitAndVeg.length == 0) {
            $scope.winCheck();
        }
    }